src/data/markdown/docs/03 cloud/03 Manage/04 Environment variables.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Environment variables'
+excerpt: 'How to use environment variables in the cloud'
+---
+
+Environment Variables are key-value pairs configured at the organization level. All values are encrypted before being stored in our database and remain encrypted until they are needed for test-runs. You reference environment variables within a test-script and the variables are interpolated on the server that runs the Test-Run. Environment Variables can be used to store sensitive (and non-sensitive) information that you want to reference in your test-scripts. Managing environment variables in one central place reduces the data exposure and ensures that the changes take effect in all test-runs that use these variables.
+
+> Changes to an Environment Variable won't apply to already started Test-Runs, they only apply to new upcoming Test-Runs.
+
+## Managing environment variables
+
+Environment Variables can be managed in Organization Settings: Select your profile icon, then **Manage > Environment variables**. Permission to create, reveal, modify and delete environment variables is restricted to Owners and Admins of the organization. 
```

Comment:
```suggestion
Only owners and admins can create, modify, and delete environment variables.
If you have approriate permission, you can manage environment variables with these steps:
1. Head to **Organization Settings**.
1. Select your profile icon, then **Manage > Environment variables**.

```

Reply:
Probably don't want to say reveal, since this could be done through logs by anyone.